babel.config.json

```diff
@@ -0,0 +1,16 @@
+{
+    "presets": [
+        "@babel/typescript"
+    ],
+    "plugins": [
+        "@babel/plugin-transform-exponentiation-operator",
+        "@babel/plugin-transform-regenerator",
+        "@babel/plugin-transform-classes",
+        "@babel/plugin-transform-object-super",
+        "@babel/plugin-transform-async-to-generator",
+        "@babel/proposal-class-properties",
+        "@babel/proposal-object-rest-spread",
```

Comment:
I think we can also drop all of those now that goja has support for practically all of them :thinking: